tests/gabbits/list-snippets-filtering.yaml

```diff
@@ -0,0 +1,131 @@
+common:
+  - &datetime_regex /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+(\+\d{2}:\d{2})|Z$/
+  - &request_id_regex /^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$/
+  - &slug_regex /^[a-zA-Z0-9]+$/
+
+fixtures:
+  - XSnippetApiWithSnippets
+
+tests:
+  - name: get recent snippets with tag=decorator
```

Comment:
This is a great idea that I did not think of before: keep the tests focused on verifying the API route we are interested in, and put the setup boilerplate into the definition of the fixture :+1: 

I think we can then revisit some other tests that I've added and clean them up in the same way.